sbin/firehol

```diff
@@ -12465,7 +12404,7 @@ then
 		then
 			# RedHat
 			FIREHOL_AUTOSAVE="/etc/sysconfig/iptables"
-		elif [ -d "/var/lib/iptables" ]
+		elif [ -d "$LOCALSTATEDIR/lib/iptables" ]
```

Comment:
as you seek for 3rd party location, this should remain /var I think, as it is not important where you actually install firehol.